### PR TITLE
fix 404 page

### LINKED ISSUE #127: [BUG] 404 is broken
Image

### PR DESCRIPTION
**Pull request by AucT**
Closes #127

404 page css for navigation is a bit different.

## Summary by CodeRabbit

- **New Features**
  - Improved the styling and presentation of the 404 error page by adding a new CSS resource.
  
- **Tests**
  - Added tests to verify the functionality and visual representation of the 404 error page in both mobile and site contexts.

### CODE REVIEW

**Comment by AucT** [comment]
5aqYocNKPd

File: themes/beaver/layouts/404.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {{ define "header" }}
-
+{{- $CSS := slice (resources.Get "css/skin-65eda28877e04.css") (resources.Get "css/586.css") (resources.Get "css/404.css") (resources.Get "css/base-4.min.css") | resources.Concat "css/single/404.css" | postCSS  | minify | fingerprint | resources.PostProcess}}
```
Comment (pftg) [inline themes/beaver/layouts/404.html]: I'd use `page/index.html` css files, which has good navigation and footer.

**Comment by AucT** [comment]
CtlJn8CoeH

**Comment by pftg** [comment]
Excellent, great work. I found some issues. Let's fix it in refactoring PR: 
jetthoughts github io – Commit: 404 png 2024-09-16 23-12-34

**Pull request merged**